features/importing/work_import.feature

```diff
@@ -220,3 +220,23 @@ Feature: Import Works
     When I import "http://www.intimations.org/fanfic/idol/Huddling.html"
     Then I should see "Preview"
       And I should see "English"
+
+  Scenario: Searching for a work url should be cached
+    Then there should be no work_url cached for "http://www.scarvesandcoffee.net/viewstory.php?sid=9570"
```

Comment:
Personally, I would rename the step to `there is no cached work_url for "<url>"`. Then the test can go: